app/javascript/packages/verify-flow/steps/personal-key/personal-key-step.tsx

```diff
@@ -16,6 +16,11 @@ function PersonalKeyStep({ value }: PersonalKeyStepProps) {
 
   return (
     <>
+      {customElements.get('lg-step-indicator') && (
+        <Alert type="success" className="margin-bottom-4">
+          {t('idv.messages.confirm')}
+        </Alert>
+      )}
```

Comment:
Another idea for having it only show up in the verify flow is maybe to render this in the parent component instead (`app/javascript/packages/verify-flow/index.tsx`)? I could imagine we eventually may have a "Flash" mechanism live there, but I think it'd be fine to just have the static confirmation message for now.

Reply:
Gotcha, yes that makes sense. I moved it there 👍